pinot-core/src/main/java/com/linkedin/pinot/core/startree/StarTreeDataTable.java

```diff
@@ -37,19 +40,34 @@
  * As a result, all fields related to memory position should be declared as long to avoid int overflow.
  */
 public class StarTreeDataTable {
+  private static final Logger LOGGER = LoggerFactory.getLogger(StarTreeDataTable.class);
+
+  private static final Int2ObjectMap<IntPair> EMPTY_INT_OBJECT_MAP = new Int2ObjectLinkedOpenHashMap<>();
+  private static final ByteOrder nativeByteOrder = ByteOrder.nativeOrder();
 
   private File file;
   private int dimensionSizeInBytes;
   private int metricSizeInBytes;
   private int totalSizeInBytes;
   final int[] sortOrder;
 
+  // Re-usable buffers
+  private LBuffer dimLbuf1;
+  private LBuffer dimLbuf2;
+  private LBufferAPI dimMetLbuf1;
+  private LBufferAPI dimMetLbuf2;
```

Comment:
I follow the convent that already exists in the file. In this file the original author did not use leading '_', so I am going to maintain the convention within the file. At some point we need to pick on convention and reformat all the code.